### PR TITLE
SONAR-8949 let api/rules/update permission check use the specified org

### PR DESCRIPTION
**Pull request by bartfastiel**

### CODE REVIEW

**Comment by SonarTech** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [UpdateAction.java#L80](https://github.com/SonarSource/sonarqube/blob/e847da536842d3551f3bb5d5905c7fb8fddaa9f6/server/sonar-server/src/main/java/org/sonar/server/rule/ws/UpdateAction.java#L80): Remove this unused "ruleWsSupport" private field. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1068)

[MAJOR]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR'

File: server/sonar-server/src/main/java/org/sonar/server/rule/ws/RuleWsSupport.java [hunk 1]
```diff
@@ -44,7 +44,7 @@ public RuleWsSupport(DbClient dbClient, UserSession userSession, DefaultOrganiza
     this.defaultOrganizationProvider = defaultOrganizationProvider;
   }
 
-  public void checkQProfileAdminPermission() {
+  public void checkQProfileAdminPermissionOnDefaultOrganization() {
```
Comment (sns-seb) [inline server/sonar-server/src/main/java/org/sonar/server/rule/ws/RuleWsSupport.java]: If I understand correctly, this renaming is not fixing the problem, it's just here to make things clearer
I agree on the change but it should be a dedicated commit

Comment (bartfastiel) [inline server/sonar-server/src/main/java/org/sonar/server/rule/ws/RuleWsSupport.java]: The change seemed too small to me to deserve a separate commit, however I'm going to keep this consideration in mind for the next changes I do.
